docs/other/payload-validation.mdx

```diff
@@ -0,0 +1,124 @@
+---
+id: payload_validation
+title: Payload validation with AMF
+---
+
+AMF can be used to validate incoming payloads to an API already deployed and running.
```

Comment:
You can use AMF to validate incoming payloads to an API already deployed and running.